lib/LaTeXML/Post/LaTeXImages.pm

```diff
@@ -32,31 +32,33 @@ our $LATEXCMD = 'latex';    #(or elatex) [ CONFIGURABLE? Encoded in PI?]
 # The purpose of this module is to convert TeX fragments into png (or similar),
 # typically via dvi and other intermediate formats.
 # LaTeX converts the TeX stuff to dvi;
-# dvips converts the dvi to eps, and ImageMagick can convert the eps to png;
+# dvips converts the dvi to ps, and ghostscript converts to png
+# then ImageMagick extracts the dimensions and converts from png if necessary
 # OR dvipng can convert the dvi to png MUCH quicker... if it's available.
+# OR dvisvgm for high quality SVG output, if available
+
+# preview.sty is used to inform gs/dvipng/dvisvgm how to clip the image
+# and recover the depth for vertical alignment
 
 # Options:
 #   source         : (dir)
 #   magnification  : typically something like 1.33333, but you may want bigger
 #   maxwidth       : maximum page width, in pixels (whenever line breaking is possible)
-#   DPI            : assumed DPI for the target medium (default 100)
+#   DPI            : assumed DPI for the target medium (default 96)
 #   background     : color of background (for anti-aliasing, since it is made transparent)
 #   imagetype      : typically 'png' or 'gif'.
 sub new {
   my ($class, %options) = @_;
   my $self = $class->SUPER::new(%options);
   $$self{magnification} = $options{magnification} || 1.33333;
   $$self{maxwidth}      = $options{maxwidth}      || 800;
-  $$self{DPI}           = $options{DPI}           || 100;
+  $$self{DPI}           = $options{DPI}           || 96;
   $$self{background}    = $options{background}    || "#FFFFFF";
   $$self{imagetype}     = $options{imagetype}     || 'png';
 
-  # Parameters for separating the clipping box from the
   # desired padding between image edge and "ink"
   $$self{padding} = $options{padding} || 2;    # pixels
```

Comment:
We may as well do
```suggestion
  $$self{padding} = $options{padding} || 0;    # pixels
```
since there is no risk of interference with framing. Or maybe one pixel to ensure antialiasing is preserved. Two pixels starts to be visible with inline formulas.